react-props-and-expressions/my-app/src/App.js

```diff
@@ -0,0 +1,20 @@
+import logo from './logo.svg';
+import './App.css';
+import './CustomButton.css';
+
+function App() {
+  return (
+    <div className="App">
+      <CustomButton text={'I'} color={'red'} />
+      <CustomButton text={'know'} color={'green'} />
+      <CustomButton text={'React!'} color={'blue'} />
```

Comment:
No `{}` needed here! We only need them when passing a variable or template literal. Since it's just a string, we can do the string directly.

`<CustomButton text="I" color="red" />`